nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/random/HunspellDictionaryHolder.java

```diff
@@ -45,13 +47,15 @@
 
     protected int wordCount;
 
-    protected String dicName;
+    protected String dicName = DEFAULT_DIC;
 
     public static final Log log = LogFactory.getLog(HunspellDictionaryHolder.class);
 
-    public HunspellDictionaryHolder(String dicName) {
+    public HunspellDictionaryHolder(String lang) {
         generator = new Random(System.currentTimeMillis());
-        this.dicName = dicName;
+        if (lang != null) {
+            dicName = lang.replaceAll("\\W+", "") + ".dic";
```

Comment:
The lang option is exposed via rest, so we need to make sure we open a dictionary file and not something else like "../../../etc/password". I will add a comment.